### PR TITLE
pull tokens from cdn instead of node_modules

### PR DESCRIPTION
**Pull request by APJohns**

### CODE REVIEW

**Review by misdreavus79 — DISMISSED** [review]
LGTM

File: .storybook/decorators.tsx [hunk 1]
```diff
@@ -7,25 +7,25 @@ interface Themes {
 
 const themesData: { [key: string]: Themes } = {
   'corporate-light': {
-    cssPath: '/node_modules/@boston-scientific/anatomy-tokens/lib/css/corporate/light.css',
+    cssPath: 'https://cdn.jsdelivr.net/npm/@boston-scientific/anatomy-tokens@5.0.0-beta.15/lib/css/corporate/light.css',
```
Comment (kaseyllabelle) [inline .storybook/decorators.tsx]: do we need to update these to .16 now? is there a way to automate this like we do in the docs site?

File: .storybook/storybook.scss [hunk 2]
```diff
@@ -1,3 +1,4 @@
+/* stylelint-disable selector-max-id */
```
Comment (kaseyllabelle) [inline .storybook/storybook.scss]: can we disable this for the declaration blocks where we know we need it vs the whole file?

File: .storybook/preview-body.html [hunk 3]
```diff
@@ -0,0 +1,4 @@
+<link
+  rel="stylesheet"
+  href="https://cdn.jsdelivr.net/npm/@boston-scientific/anatomy-tokens@5.0.0-beta.15/lib/css/corporate/light.css"
```
Comment (kaseyllabelle) [inline .storybook/preview-body.html]: do we need .16 here too?

**Review by kaseyllabelle — APPROVED** [review]

**Pull request merged**